README.md

```diff
@@ -4,6 +4,9 @@ This is a programmatic interface in Ruby for SignalFx's metadata and
 ingest APIs. It is meant to provide a base for communicating with
 SignalFx APIs that can be easily leveraged by scripts and applications
 to interact with SignalFx or report metric and event data to SignalFx.
+You will need a SignalFx account and organization API token to use them.
```

Comment:
It seems weird that we are telling them how to get an account. We don't do that anywhere else. Can this be consistent with other docs?